layouts/partials/implementations.html

```diff
@@ -26,7 +25,7 @@ <h5 class="content-center">JavaScript</h5>
       </div>
     </div>
 
-    <h2 id="integrations-title" class="content-center">Integrations</h2>
+    <h2 id="integrations-title" class="content-center">Apps</h2>
```

Comment:
In a distant past it was `Apps`, but @lidel suggested changing it https://github.com/ipfs/website/pull/282#pullrequestreview-182658539 and it stayed that way.

Reply:
I will wrestle @lidel for it.

Reply:
😱🥊

Other than that it's good to go 👍

Reply:
Now that I think about it, `Apps` is fine :face_with_head_bandage: